CMakeLists.txt

```diff
@@ -8,6 +8,10 @@ endif()
 
 set(BRPC_VERSION 0.9.0)
 
+SET(CPACK_GENERATOR "DEB")
+SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "Baidu")
```

Comment:
Please change the maintainer to "brpc authors"

Reply:
Done.